content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Accessing HPE Data Fabric S3 Storage from Spring Boot S3 Micro Service
+  deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: /img/hpe-dev-grommet-gremlin-rockin.svg
+tags:
```

Comment:
Please use hpe-ezmeral-data-fabric and mapr as tags so it gets associated with corresponding platform page

Reply:
done

Reply:
Hi Kiran,

Could you please change the tag "HPE Ezmeral Data Fabric" into "hpe-ezmeral-data-fabric" (with dash between each word in lowercase). This will allow the system to automatically associate your blog post to existing "platform": HPE Ezmeral Data Fabric.